python/ct/crypto/cert.py

```diff
@@ -378,21 +382,33 @@ def subject_alternative_names(self):
         sans = self._get_decoded_extension_values(oid.ID_CE_SUBJECT_ALT_NAME)
         return sum([list(san) for san in sans], [])
 
-    def subject(self):
-        """Returns list of subject field values in fashion similar to
-        subject_alternative_names fashion.
+    def subject(self, flatten=True):
```

Comment:
The change here is for a very specific use case so I recommend leaving it out of the general API.
Instead, get the value from the asn1 the in verify.py - the same way you do for the other extensions.